### DISCUSSION TITLE
Receiving "Error creating app"

### DISCUSSION THREAD

**Discussion by Mubo507** [Budibase/budibase#7074] [Help]
I am receiving that error every time I try to create an app or use a template. I am seeing this error in docker, could this be the issue?
![image](https://user-images.githubusercontent.com/110569958/182712909-65b59102-0e53-431a-8ff2-39eeeb333fa5.png)

**Comment by mike12345567** [comment]
Hi @Mubo507 - are you using the Docker single image?

Can you explain how you've run the image, are you using Docker-compose or are you running it directly with the run command?

**Comment by YariKoen** [comment]
Hello, I am getting the same errors and I am using app in browser;

- when I try to create app from scratch or template > `Error creating app`
- when I reload page, apps are there; but after click on them > `Error initializing app: [object Object]`
- when I try to delete these apps > `Error deleting app`

I tried to run it in Incognito, so it should not be client-side problems. I also got some errors during sign up, but not related to this topic :)

**Reply by shogunpurple** [reply]
Hi @YariKoen, more information here - https://github.com/Budibase/budibase/issues/7395#issuecomment-1221994154
